spec/requests/api/alert_definitions_spec.rb

```diff
@@ -143,3 +144,115 @@
     expect(alert_definitions.second.reload.description).to eq("Updated Test Alert 2")
   end
 end
+
+describe "Alerts Definition Profiles API" do
+  it "reads 2 alert definition profiles as a collection" do
```

Comment:
let's add a test for accessing without appropriate role and expect the forbidden error.  Thanks.